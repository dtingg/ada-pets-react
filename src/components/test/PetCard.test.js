import React from "react";
import { render, cleanup } from "@testing-library/react"
import PetCard from "../PetCard";


describe("PetCard", () => {
  test("Snapshot test for PetCard", () => {
    const container = render(
    <PetCard
      id={1}
      name={"Dino"}
      species={"dog"}
      location={"Seattle"}
      images={["placekitten.com"]}
      about={"New pet"}
      selectPet={() => {}}
      removePet={() => {}}
    />);

    expect(container.asFragment()).toMatchSnapshot();
    cleanup();
  });
});
