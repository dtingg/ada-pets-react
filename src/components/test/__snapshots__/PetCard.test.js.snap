// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PetCard Snapshot test for PetCard 1`] = `
<DocumentFragment>
  <div
    class="card pet-card"
  >
    <section
      class="pet-card--header"
    >
      🐕 1 - Dino
      <button
        class="btn btn-primary pet-card--select-pet-btn"
      >
        Select
      </button>
      <button
        aria-label="Remove"
        class="btn btn-danger pet-card--remove-btn"
        type="button"
      >
        Remove
      </button>
    </section>
    <section
      class="pet-card--body"
    >
      New pet
    </section>
    <section
      class="pet-card--footer text-muted"
    >
      Seattle
    </section>
  </div>
</DocumentFragment>
`;
